README.md

```diff
@@ -3,19 +3,61 @@ Clapper
 
 Synchronize setup and deployment for the Director.
 
-Configuration Validation
-========================
+Ansible-based Validations
+-------------------------
+
+Recently, we focused on writing automated validations that require
+little human setup or interference. These are driven by
+[Ansible](http://ansible.com/) we plan to provide an API that will let
+us use these from the TripleO web UI and command line clients.
+
+### Running a validation
+
+For now, you need to run them manually. All the validations live in
+the [ansible-tests/validations](ansible-tests/validations) directory.
+
+To run a validation you need to:
```

Comment:
Right. On the other hand, I'm not keen on telling people to install Flask and everything else when they don't need it -- especially on production servers.

How about we split the requirements.txt for pure ansible and the API? And we'll remove the latter + docs when we switch to mistral.